src/Microsoft.Azure.WebJobs.Host/Hosting/LoggableOptions/ILoggableOptionsWrapper.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Hosting
+{
+    /// <summary>
+    /// Allows framework types to be registered as LoggableTypes.
+    /// </summary>
+    /// <typeparam name="TOptions">The options type.</typeparam>
+    internal interface ILoggableOptionsWrapper<TOptions> : ILoggableOptions where TOptions : class, new()
```

Comment:
I should change this to have it be `GetLoggableOptions(TOptions options)` rather than a two-step set-then-get.